config/app-setup/vars.json

```diff
@@ -0,0 +1,104 @@
+{
```

Comment:
I'm not sure if this is possible, but I would prefer this to be imported in the base config just like all the other imports that we do there (and then put this file in something like `config/app/variables.default.json`), except I'm not completely sure if this is possible.

@rubensworks is components.js smart enough to ignore all the triples that are not relevant for building a component? To explain what the code does here, first it builds `urn:solid-server-app-setup:default:VarResolver` and it uses the output of that object to have values for the variables when building `urn:solid-server:default:App`. So my question is if Components.js would have an issue if both of these are in the same config when first building `VarResolver`? The issue is that the variables would not have values yet, but none of the classes that depend on them should be instantiated in the first execution. Also not sure if this would have a performance impact. Maybe there are other disadvantages I'm missing.

This would also allow us to remove the `config/util/variables` setting and add that behaviour here so we don't have 2 places where variables are (sort of) defined.

Reply:
I don't fully understand the issue, but in any case, Components.js will only instantiate those components that are needed. So if you include components in your config, but they are not needed for an instantiation at runtime, they will just be ignored.